src/app/rpmostree-builtin-status.c

```diff
@@ -48,24 +53,42 @@ static GOptionEntry option_entries[] = {
   { NULL }
 };
 
+static const char *bold_prefix = "";
+static const char *bold_suffix = "";
+static const char *red_prefix = "";
+static const char *red_suffix = "";
+
+static gboolean
+is_tty ()
```

Comment:
`(void)` - remember this is C so `()` doesn't mean what one would expect.

Actually why not just use the `is_tty` field of `GLnxConsole`?